features/importing/work_import_da.feature

```diff
@@ -11,50 +11,35 @@ Feature: Import Works from deviantart
       And I fill in "urls" with "http://bingeling.deviantart.com/art/Flooded-45971613"
     When I press "Import"
     Then I should see "Preview"
-      And I should see the image "src" text "Flooded_by_bingeling.jpg"
+      And I should see the image "src" text "http://orig03.deviantart.net/4707/f/2007/004/a/7/flooded_by_bingeling.jpg"
       And I should see "Digital Art" within "dd.freeform"
       And I should see "People" within "dd.freeform"
       And I should see "People" within "dd.freeform"
```

Comment:
Looks as if there's a duplicate line in the original test - might as well take this opportunity to remove it